package.json

```
@@ -18,23 +18,25 @@
     "release:major": "standard-version --release-as major",
     "lint": "standard",
     "lint:fix": "standard --fix",
-    "test": "echo \"Info: no test specified\" && exit 0"
+    "test": "echo \"Info: no test specified\" && exit 0",
+    "ncu": "node_modules/.bin/ncu"
```

We can use the npx version and avoid loading the dependency in the project:

```suggestion
    "deps": "npx npm-check-updates",
    "deps:upgrade": "npx npm-check-updates -- -u",
```

It will be good also to include it in the Github Actions in order to ensure that the dependecies are checked 🤔